marshal.go

```diff
@@ -136,8 +139,14 @@ func Marshal(info TypeInfo, value interface{}) ([]byte, error) {
 	}
 
 	switch info.Type() {
-	case TypeVarchar, TypeAscii, TypeBlob, TypeText:
-		return marshalVarchar(info, value)
+	case TypeVarchar:
+		return marshalVarchar(value)
+	case TypeText:
+		return marshalText(value)
+	case TypeBlob:
+		return marshalVarchar(value)
```

Comment:
why having marshalBlob function if you don't you it here?

Reply:
It`s my mistake.
Fixed.